test/e2e/storage/external/external_test.go

```diff
@@ -34,7 +35,10 @@ func TestDriverParameter(t *testing.T) {
 			),
 		},
 		ShortName: "foo",
-		ClaimSize: "5Gi",
+		SupportedSizeRange: volume.SizeRange{
+			Min: "1Mi",
+			Max: "10Gi",
```

Comment:
ditto

Reply:
Have reset to original size as suggested 5Gi